src/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>MB's Place</title>
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+        integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+
+</head>
+
+<body>
+    <header>
+        <nav>
+            <ul class="nav-list color-coffee">
+                <li>MB's CookBook</li>
+                <li><a class="nav" href="#">Recipes (<span id="recipeCount"></span>)</a></li>
+                <li><a class="nav" href="#">Checkout Meals</a></li>
+                <li><a class="nav" href="#">Contact Us</a></li>
+                <li><input type="text" id="input" placeholder="Enter first letter of dish"></li>

```

Comment:
- [ ] <b>OPTIONAL:</b>This is not in project requirements but I highly recommend you to consider this:
   -  For all websites and web apps, logos are always links that lead to the homepage. in your case, you are supposed to wrap `MB's CookBook` in anchor tag `<a>` with property `href='homepage link'`.
   - It is not good to have links and inputs that are not functional for a complete application.